apps/remix-ide/src/app/tabs/test-tab.js

```diff
@@ -74,11 +74,21 @@ module.exports = class TestTab extends ViewPlugin {
     }
   }
 
+  getTestlibs () {
+    return { assertLibCode, accountsLibCode: this.testRunner.accountsLibCode }
+  }
+
   async onActivation () {
     const isSolidityActive = await this.call('manager', 'isActive', 'solidity')
     if (!isSolidityActive) {
       await this.call('manager', 'activatePlugin', 'solidity')
     }
+    await this.testRunner.init()
+    const provider = await this.fileManager.getProviderOf()
```

Comment:
can you use `currentFileProvider()` instead?

Reply:
yes I can